File: hello_world.py
#! /usr/bin/env python3
__module__="hello_world"
__author__="CyberError"
## Example of Hello World program in python 3.
print("Hello World")
##
## A little bit more advanced re-usable hello world:
def hello_world(name="World"):
    print("Hello %s")

hello_world()
name=input("Your name please?")
hello_world(name)

File: hello_world.php
<!DOCTYPE html>
<!-- output html for the document -->
<html>
<head>
<meta name="author" content="CyberError"/>
<?php 
/*
 * Example of Hello World in php.
 * @author CyberError
 */
// Form for inputing a name:
function output_html($name){
echo<<<END
<title>Hello $name</title>
</head>
<body>
<form method="GET">
<input type="text" name="name" placeholder="Name" />
<input type="submit" />
</form>
END;
}
// Is name being input?
if(isset($_GET['name']) or isset($_POST['name'])){
@$name = $_GET['name'] ?: $_POST['name'];
output_html($name);
echo("Hello ".$name);
}else{
// If not echo Hello World
output_html("World");
echo("Hello World");
}
?>
</body>
</html>	
